runtime/sema/authaccount.cdc

```diff
@@ -338,6 +338,14 @@ pub struct AuthAccount {
         /// Returns the capability if one was published at the path.
         /// Returns nil if no capability was published at the path.
         pub fun unpublish(_ path: PublicPath): Capability?
+
+        /// **DEPRECATED**: This function only exists temporarily to aid in the migration of links.
+        /// This function will not be part of the final Capability Controller API.
+        ///
+        /// Migrates the link at the given path to a capability controller.
+        ///
+        /// Does not migrate intermediate links of the chain.
```

Comment:
Maybe add a brief description on what the return value is.

Reply:
Added in fc6436f0a1f83bbffc7e44502f5102aa8f95ceb1